### ISSUE TITLE
Data linking only works to 2 levels

### ISSUE THREAD

**Issue by SamuraiDante** [jerosoler/Drawflow#169]
df-level1-level2-value works
df-level1-level2-level3-value seemingly does not.

Maybe I'm doing it wrong, first one works so I'm using it.
Not an issue for me, just thought I'd let you know.

**Comment by jerosoler** [comment]
Thanks! @SamuraiDante 

You're right!
I have tried the following code and I see that it does not work, I will check it.
```javascript 
          const data = { 
           level1: {
            name: 'TEST 1',
            level2: {
              name: 'test 2',
              level3: {
                name: "TESTTTTT 3"
              }
            }
           }
          }
          const htmltest = `
          <input type="text" df-level1-name>
          <input type="text" df-level1-level2-name>
          <input type="text" df-level1-level2-level3-name>
          `;
          editor.addNode('test2', 1,1, 100, 200, 'test2', data, htmltest );
```
Jero

**Comment by jerosoler** [comment]
Fix in version 0.0.41

**Comment by SamuraiDante** [comment]
That was fast!
